### PR TITLE
1.5.1 : New Japanese Apple Podcast User Agent & Update to OS mappings

### PR DESCRIPTION
**Pull request by liambly**
Adds support for the Apple Podcasts ポッドキャスト(poddokyasuto) user agent via additional regex in `apple_podcasts.rb`. Additionally, updates the Operating system mappings `lib/user_agent/operating_systems/*`.

Proposes gem version 1.5.0 -> 1.5.1.

### CODE REVIEW

File: lib/user_agent/operating_systems/darwin.rb
Comment (hungyunliao) [inline lib/user_agent/operating_systems/darwin.rb]:
```suggestion
        '22.0.0' => '16.0.x',
```
Probably it's better to use `16.0.x`? You will also need to change your spec accordingly.

File: lib/user_agent/operating_systems/darwin.rb
Comment (hungyunliao) [inline lib/user_agent/operating_systems/darwin.rb]:
```suggestion
        '21.5.0' => '15.5',
```
Nitpick: the comment `# also 15.5.1` doesn't seem right. From the [table](https://www.theiphonewiki.com/wiki/Kernel#iOS.2FiPadOS) (`iOS/iPadOS`) there is no `15.5.1` being mapped. `15.5.1` mapping is for `tvOS`.

File: lib/user_agent/operating_systems/darwin.rb
Comment (hungyunliao) [inline lib/user_agent/operating_systems/darwin.rb]:
```suggestion
        '21.6.0' => '15.6',  # also 15.6.1, 15.7, 15.7.1
```

File: lib/user_agent/operating_systems/darwin.rb
Comment (hungyunliao) [inline lib/user_agent/operating_systems/darwin.rb]:
```suggestion
        '22.0.0' => '9.0.x',
```

File: spec/browsers/apple_podcasts_user_agent_spec.rb
Comment (hungyunliao) [inline spec/browsers/apple_podcasts_user_agent_spec.rb:162]:
```suggestion
  let!(:useragent) { UserAgent.parse("%E3%83%9D%E3%83%83%E3%83%89%E3%82%AD%E3%83%A3%E3%82%B9%E3%83%88/1709.2.2 CFNetwork/1390 Darwin/22.0.0") }
  
```
Super nitpick for styling. Just to add a blank line to be consistent with the above.

File: spec/browsers/apple_podcasts_user_agent_spec.rb
Comment (hungyunliao) [inline spec/browsers/apple_podcasts_user_agent_spec.rb:168]:
```suggestion
  let!(:useragent) { UserAgent.parse("ポッドキャスト/1709.2.2 CFNetwork/1390 Darwin/22.2.0") }
  
```

**Review by hungyunliao — COMMENTED** [review]
Just some nitpicks. Looks good to me!

File: lib/user_agent/operating_systems/darwin.rb
Comment (liambly) [reply lib/user_agent/operating_systems/darwin.rb]: Thoughts on having a comment trailing the line? : `'22.0.0' => '16.0.x', # 16.0, 16.0.1, 16.0.2 and 16.0.3`

File: lib/user_agent/operating_systems/darwin.rb
Comment (liambly) [reply lib/user_agent/operating_systems/darwin.rb]: Thoughts on having a comment trailing the line? : `'22.0.0' => '9.0.x', # 9.0, 9.0.1 and 9.0.2`

File: lib/user_agent/operating_systems/darwin.rb
Comment (hungyunliao) [reply lib/user_agent/operating_systems/darwin.rb]: I would prefer without the comment just to be consistent with the others, but I think either is fine. Thanks!

File: lib/user_agent/operating_systems/darwin.rb
Comment (hungyunliao) [reply lib/user_agent/operating_systems/darwin.rb]: Same here.

Comment (liambly) [reply lib/user_agent/operating_systems/darwin.rb]: Will remove - thanks!

File: lib/user_agent/operating_systems/darwin.rb
Comment (liambly) [reply lib/user_agent/operating_systems/darwin.rb]: Will remove - thanks!

**Review by hungyunliao — APPROVED** [review]
This looks good to me. Usually a PR needs two approvals before merging so please also make sure Keith reviews again and approves the final version. Thank you.

File: useragent.gemspec
Comment (ziggythehamster) [inline useragent.gemspec:22]:
```suggestion
  s.email   = ['garry@robustsoftware.co.uk', 'keith@art19.com', 'kit@art19.com', 'liambly@amazon.com', 'louise@art19.com', 'johannes@art19.com']
  
```

**Review by ziggythehamster — CHANGES_REQUESTED** [review]
Can you also change .travis.yml so it tests in Ruby 3.0 and 3.1?

File: useragent.gemspec
Comment (liambly) [reply useragent.gemspec:22]: Added newline

**Comment by liambly** [comment]
@ziggythehamster Added Ruby 3.0 and 3.1 to .travis.yml.

**Review by ziggythehamster — APPROVED** [review]

**Pull request merged**
